src/tests/components/ForecastSummaries.test.js

```diff
@@ -1,51 +0,0 @@
-import React from "react";
-import { render } from "@testing-library/react";
-import ForecastSummaries from "../../components/ForecastSummaries";
-
-describe("Forecast Summaries", () => {
-  const validProps = {
-    forecasts: [
-      {
-        date: 1111111,
-        description: "Stub description1",
-        icon: 800,
-        temperature: {
-          min: 12,
-          max: 22,
-        },
-      },
-      {
-        date: 2222222,
-        description: "Stub description2",
-        icon: 602,
-        temperature: {
-          min: 24,
-          max: 13,
-        },
-      },
-    ],
-    onForecastSelect: () => {},
-  };
-
-  test("renders correctly", () => {
-    const { asFragment } = render(
```

Comment:
you can destructure here as well:
`const { forecasts, onForecastSelect } = validProps`